commands/commands.go

```diff
@@ -104,13 +104,26 @@ var Commands = []cli.Command{
 				Name:  "rate", // We default to the rate per minute. Please provide this rate in req/min to make.
 				Value: 15,
 			},
+			cli.StringFlag{
+				Name:  "logout",
+				Value: "log.txt",
+			},
 			cli.StringFlag{
 				Name:  "out",
-				Value: "data/data-test.txt",
+				Value: "",
+			},
```

Comment:
Do we want to continue maintaining `out` in addition to `logout`?

Reply:
This is the output file for writing the JSON-encoded results, not the log output.

Reply:
Ah missed that, thank you, this is cleaner!